packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3218,7 +3218,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       }
       assert(composingRect != null);
       _textInputConnection!.setComposingRect(composingRect);
-      SchedulerBinding.instance.addPostFrameCallback((Duration _) => _updateComposingRectIfNeeded());
+      Future<void>.delayed(const Duration(milliseconds: 250), () => _updateComposingRectIfNeeded());
```

Comment:
I think this could lead to crash? The text in the input field can be changed changed between frames and `TextPainter` will probably crash when  `_updateComposingRectIfNeeded` queries invalidated text layout.

Reply:
I can switch it around to still be tied to `addPostFrameCallback` as long as you think that dropping the frequency to 4hz is a good idea.

Reply:
I switched it so that the delay adds a post frame callback.  That addresses your concern, right?